Cargo.toml

```diff
@@ -1,4 +1,6 @@
 [workspace]
+resolver = "1"
```

Comment:
Unless things fail with the new resolver mode, I would use "2" here.

Reply:
Yeah, `sbt` fails - didn't dig into it to unblock the other PRs. :)